src/components/searchInput/MainSearchInput.jsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { Cancel, Search } from "@material-ui/icons";
+import { reduxForm, Field, reset } from "redux-form";
+import "./MainSearchInput.scss";
+import propTypes from "prop-types";
+import { connect } from "react-redux";
+import PLACEHOLDER_INPUT from "../../constants/HeadArticleSearchConstants";
+import debounce from "../../services/Debaunce";
+
+const SearchInput = ({ searchFunc, searchText, name, clear, resetForm }) => {
+  const onTitleChange = value => {
+    debounce(() => searchFunc(value), 800)();
```

Comment:
move `800` to debounce func as default value for delay. Create const for it and reuse

Reply:
fixed